oat-sa/extension-tao-test-runner-plugins
# Feature/tr 1057 remove ie11 fulscreen behavior

Related to https://oat-sa.atlassian.net/browse/TR-1057

- remove ie11 specific fullscreen behavior
- FYI, this was the PR what I had to revert https://github.com/oat-sa/extension-tao-test-runner-plugins/pull/184/files

Test:
- install this version of extension
- have a delivery, where security is enabled
- check fullscreen in ie11 (do not forget to disable debug mode)
- when test is loaded you should see `This test needs to be taken in full screen mode (F11).` message instead of `The assessment must be taken in fullscreen. Please press F11 to activate it.`
